corptools/src/components/CharAssetLocSelect.js

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect, useState } from "react";
+import { useQuery } from "react-query";
+import Select from "react-select";
+import { loadAssetLocations } from "../apis/Character";
+
+const CharAssetLocSelect = ({ character_id, setLocation }) => {
+  const [location_id, setLoc] = useState(0);
```

Comment:
You should be able to pass down the value for location id from the parent, getting rid of this `useState`